### PR TITLE
increase test coverage of mocha.js

### PR DESCRIPTION
**Pull request by seppevs**
Adds a few tests & increases the test coverage of mocha.js

### CODE REVIEW

File: test/mocha.spec.js [hunk 1]
```diff
@@ -32,4 +32,133 @@ describe('Mocha', function () {
...
+  });
+
+  describe('.ignoreLeaks()', function () {
+    it('should set the ignoreLeaks option to true when param equals true', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.ignoreLeaks(true);
+      mocha.options.ignoreLeaks.should.equal(true);
+    });
+
+    it('should set the ignoreLeaks option to true when param equals false', function () {
```
Comment (ScottFreeCode) [inline test/mocha.spec.js]: "true" here should read "false".

File: test/mocha.spec.js [hunk 2]
```diff
@@ -32,4 +32,133 @@ describe('Mocha', function () {
...
+      var mocha = new Mocha(blankOpts);
+      mocha.ignoreLeaks(false);
+      mocha.options.ignoreLeaks.should.equal(false);
+    });
+
+    it('should be chainable', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.ignoreLeaks(false).should.equal(mocha);
+    });
+  });
```
Comment (ScottFreeCode) [inline test/mocha.spec.js]: Do we want to also test the behavior of `.ignoreLeaks()` (no parameter) while we're at it?

File: test/mocha.spec.js [hunk 3]
```diff
@@ -32,4 +32,133 @@ describe('Mocha', function () {
...
+
+  describe('.useInlineDiffs()', function () {
+    it('should set the useInlineDiffs option to true when param equals true', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.useInlineDiffs(true);
+      mocha.options.useInlineDiffs.should.equal(true);
+    });
+
+    it('should set the useInlineDiffs option to true when param equals false', function () {
```
Comment (ScottFreeCode) [inline test/mocha.spec.js]: "true" vs. "false" again

**Comment by seppevs** [comment]
Changes done.

**Pull request merged**